diff --git a/vendor/raylib/src/platforms/rcore_desktop_glfw.c b/vendor/raylib/src/platforms/rcore_desktop_glfw.c
index 55b92ac7..b45cd1cb 100644
--- a/vendor/raylib/src/platforms/rcore_desktop_glfw.c
+++ b/vendor/raylib/src/platforms/rcore_desktop_glfw.c
@@ -2012,6 +2012,7 @@ static void WindowMaximizeCallback(GLFWwindow *window, int maximized)
 {
     if (maximized) FLAG_SET(CORE.Window.flags, FLAG_WINDOW_MAXIMIZED);  // The window was maximized
     else FLAG_CLEAR(CORE.Window.flags, FLAG_WINDOW_MAXIMIZED);          // The window was restored
+    glfwGetWindowPos(window, &CORE.Window.position.x, &CORE.Window.position.y);
 }
 
 // GLFW3: Window focus callback, runs when window get/lose focus
